packages/lib/src/components/Sepa/Sepa.tsx

```diff
@@ -15,6 +15,11 @@ class SepaElement extends UIElement {
         showFormInstruction: true
     };
 
+    constructor(props) {
+        super(props);
+        this.state = { ...this.state, ...{ data: { ibanNumber: '', ownerName: '' } } };
```

Comment:
I haven't tested it locally, but isn't the state already initialized in the `IbanInput.tsx` constructor?

Reply:
That's the problem though. It is, (already initialized in the `IbanInput.tsx` constructor) but it doesn't propagate until the shopper interacts with the component

Reply:
I see, it looks like the `IbanInput.tsx` only propagates to `Sepa.tsx` when `if (this.state.data['ibanNumber'] || this.state.data['ownerName'])`. But i think let the `Sepa.tsx` decides when to propagate is better, with less responsibility for the shared component - `IbanInput.tsx`.